### PR TITLE
feat(bundle): exclude policies from full zip

### LINKED ISSUE #17: Do not include policies bundle into full bundle
Because policies are already into the plugins directory, we don't have to provide the policy bundle with the full zip.

### PR DESCRIPTION
**Pull request by NicolasGeraud**
fix #17

### CODE REVIEW

File: src/main/python/package_bundles.py [hunk 1]
```diff
@@ -215,7 +214,7 @@ def prepare_mgmt_bundle(mgmt):
 def prepare_policies(version):
     print("==================================")
     print("Prepare Policies")
-    policies_dist_path = "%s/dist/graviteeio-policies-%s" % (tmp_path, version)
+    policies_dist_path = "%s/dist/gravitee-policies-%s" % (tmp_path, version)
```
Comment (brasseld) [inline src/main/python/package_bundles.py:217]: Why did you rename dist from graviteeio-xxxxx to gravitee-xxxxx ?

Comment (NicolasGeraud) [inline src/main/python/package_bundles.py:217]: it's a little bit weird I know, but later on the script I rename directories (because we have named distributions with gravitee-...

Comment (NicolasGeraud) [inline src/main/python/package_bundles.py:217]: but perhaps I could rename gravitee- to graviteeio- . So I could keep policies_dist_path named "graviteeio-policies-XXX"

WDYT ?

Comment (brasseld) [inline src/main/python/package_bundles.py:217]: :+1: , it was just for my understanding
